snowplowtracker-scm-1.rockspec

```diff
@@ -1,7 +1,8 @@
 package = "SnowplowTracker"
-version = "0.2.0-1"
+version = "scm-1"
 source = {
-  url = "http://snowplow-hosted-assets.s3.amazonaws.com/1-trackers/lua-tracker/snowplowtracker-0.1.0.tar.gz"
+  url = "git://github.com/snowplow/snowplow-lua-tracker",
+  tag = "SET_BY_GITHUB_ACTIONS_DO_NOT_MANUALLY_SET"
```

Comment:
Just a few lines below and I think I got an answer to my question 😅  So scm means that the version is taken from this source.tag attribute that is set by GH actions from the git tag?